### ISSUE TITLE
[Search v1] - Deleted expense is not removed from Search

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#43415]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.81-1
**Reproducible in staging?:** y
**Reproducible in production?:** no
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** applause internal team
**Slack conversation:**

## Action Performed:
Precondition:
- User has many expenses.

1. Go to staging.new.expensify.com
2. Go to workspace chat.
3. Submit an expense.
4. Go to Search.
5. Note that the submitted expense is shown.
6. Go to workspace chat and delete the expense.
7. Refresh the page.
8. Go to Search.

## Expected Result:
The deleted expense will be removed from Search.

## Actual Result:
The deleted expense is not removed from Search.
This issue is not reproducible with new account that doesn't have expenses.
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/38435837/a0446077-cc4f-47dd-878b-bc1df1b58ead

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @luacmartins</details>

**Comment by m-natarajan** [comment]
@puneetlath FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by m-natarajan** [comment]
We think that this bug might be related to #wave-collect - Release 1

**Comment by luacmartins** [comment]
This is expected for now.

**Comment by luacmartins** [comment]
Closing as a dupe of https://github.com/Expensify/App/issues/43382

**Comment by puneetlath** [comment]
@luacmartins will you be tackling this yourself?

**Comment by luacmartins** [comment]
I think this is internal. Ideally we find a volunteer to tackle this, otherwise I can take a look as a lower priority item.

**Comment by luacmartins** [comment]
Still waiting on volunteer

**Comment by luacmartins** [comment]
Still no volunteers. I'll look into this one later this week.

**Comment by luacmartins** [comment]
Same as above.

**Comment by luacmartins** [comment]
No updates yet. I'll start working on it this week.

**Comment by luacmartins** [comment]
Prioritizing other Search v2 issues

**Comment by luacmartins** [comment]
Prioritizing other Search v2 issues

**Comment by luacmartins** [comment]
Prioritizing other Search v2 issues

**Comment by luacmartins** [comment]
Prioritizing other Search v2 issues

**Comment by luacmartins** [comment]
Prioritizing other Search v2 issues

**Comment by luacmartins** [comment]
PR in review

**Comment by luacmartins** [comment]
This is done
